lib/hbs-helpers.js

```diff
@@ -5,14 +5,14 @@ module.exports.register = function (handlebars) {
 	/**
 	 * defines a default variable value, that gets overwritten by context data
 	 */
-	handlebars.registerHelper('def', function (a, b) {
+	handlebars.registerHelper('bioHelperDef', function (a, b) {
```

Comment:
hi @smohadjer sorry for that change, can we use bio without helper? Think bioInclude, bioDef sounds and types much better. sorry again